benchmark/run.py

```diff
@@ -0,0 +1,181 @@
+"""Performance comparison between Helion, torch.compile, Triton, and PyTorch eager by leveraging TritonBench.
+
+Currently supported kernels are in `benchmark/`.
+
+Usage:
+$ python run.py [tritonbench args...] --kernel <kernel_name>
+
+Example usage:
+$ python benchmark/run.py --metrics speedup,accuracy --kernel vector_add
+"""
+
+from __future__ import annotations
+
+import argparse
+import importlib
+import os
+import subprocess
+import sys
+from typing import Any
+from typing import Callable
+
+# Maps tritonbench op names to Helion kernel examples
+KERNEL_MAPPINGS: dict[str, tuple[str, str]] = {
+    # <tritonbench_op_name>: (<helion_kernel_module_path>, <helion_kernel_function_name>)
+    "vector_add": ("examples.add", "add"),
+}
+
+
+def check_and_setup_tritonbench() -> None:
+    """Check if tritonbench is properly initialized and installed."""
+    helion_kernel_dir = os.path.dirname(os.path.abspath(__file__))
+    helion_root = os.path.abspath(os.path.join(helion_kernel_dir, ".."))
+    tritonbench_path = os.path.join(helion_root, "third_party/tritonbench")
+
+    # Check if tritonbench directory exists and has content
+    if not os.path.exists(tritonbench_path) or not os.listdir(tritonbench_path):
+        print("Tritonbench submodule not initialized. Initializing and installing...")
+        try:
+            # First, initialize submodule
+            subprocess.run(
+                ["git", "submodule", "update", "--init", "--recursive"],
+                cwd=helion_root,
+                check=True,
+            )
+
+            # Then run install.py
+            original_dir = os.getcwd()
+            os.chdir(tritonbench_path)
+            subprocess.run([sys.executable, "install.py"], check=True)
+            os.chdir(original_dir)
+
+            print("Tritonbench setup completed successfully.")
+
+        except subprocess.CalledProcessError as e:
+            print(f"Error setting up tritonbench: {e}")
+            sys.exit(1)
+        except Exception as e:
+            print(f"Unexpected error during setup: {e}")
+            sys.exit(1)
+
+    # Add to path
+    if tritonbench_path not in sys.path:
+        sys.path.insert(0, tritonbench_path)
```

Comment:
Modifying the path is a bit hacky.  Could we do `pip install git+https://github.com/pytorch-labs/tritonbench.git` or `(cd tritonbench && pip install -e .)`?  (If the git one works we wont need a submodule.)